brainstormforce/ultimate-addons-for-gutenberg
# AST-3382 - Custom class to anchor links of TOC to get distinguish on webpage for further exclusion

### Description
- Adding a custom class to TOC anchors for excluding in smooth-scroll Astra JS to resolve jerk issue on anchor clicks.
- AST ref PR - https://github.com/brainstormforce/astra/pull/5297

### Screenshots
- https://d.pr/v/HM9AYC (this is tab module video, getting similar for TOC)
- https://secure.helpscout.net/conversation/2236457040/386697?folderId=6758877

### Types of changes
- Bug fix 

### How has this been tested?
- Get TOC module on page
- Enable "Smooth Scroll to ID" from Astra customizer > Global > Misc
- Check on the frontend
- The jerk should not be there

### Checklist:
